index.html

```diff
@@ -309,14 +309,14 @@ <h1 class="greet">
           <ul class="skills">
               <ul class="languages"> 
                 <li><p>Languages</p></li>
-                <li class="arrow"><img src="assets/Unionlangs.png"></li>
+                <li class="arrow"><img src="assets/Unionlangs.png" alt=""></li>
               </ul>
               <div class="langs_list">
               <li class="js_2"> 
-                <img src="assets/jsLogo.png"><p>Javascript</p>
+                <img src="assets/jsLogo.png" alt=""><p>Javascript</p>
               </li>
               <li class="Html_2">
-                <img src="assets/htmlLogo.png"><p>Html</p>
+                <img src="assets/htmlLogo.png" alt=""><p>Html</p>
```

Comment:
- [ ] Same here

Reply:
Same thing I mentioned earlier 😉 the `alt` (short for alternative) is just a safeguard in case the HTML logo is not loaded correctly, which might happen in case of connection issues. 🕸️